leeanp.php

```diff
@@ -24,3 +24,6 @@
 // Run the plugin setup.
 $setup = new PluginSetup();
 register_activation_hook( __FILE__, [ $setup, 'maybe_deactivate' ] );
+
+// Init ECommerce endpoints.
+add_action( 'rest_api_init', [ $setup, 'init' ] );
```

Comment:
I think those files and code is required: 

- https://github.com/moxie-lean/wp-woocommerce-api/blob/395c6a04c18011778372bcdff5c25d333409e2f7/src/Modules/Bootstrap.php
- https://github.com/moxie-lean/wp-woocommerce-api/blob/395c6a04c18011778372bcdff5c25d333409e2f7/PluginSetup.php#L16-L21

This can be replaced by: 

```php
add_action( 'rest_api_init', function(){
  CartEndpoint::init();
  …
});
```